CMakeLists.txt

```diff
@@ -102,6 +102,9 @@ if(NO_MSG_HANDLER)
    add_definitions(-DNO_MSG_HANDLER=1)
 endif()
 
+set(OC_EXTRA_PLUGIN_PATH "" CACHE STRING "Extra path to look for Qt plugins")
+add_definitions(-DOC_EXTRA_PLUGIN_PATH=${OC_EXTRA_PLUGIN_PATH})
```

Comment:
Please don't do it like this. If you absolutely want to use -D instead of putting it in a config.h file, then only apply it to the relevant target not to all.

Reply:
We should probably also define a sane default that just works.

Reply:
I'll make these changes. What default value would you suggest? `/usr/lib/owncloud-client`? I imagine it might be different per distro.